<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="tempText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="weatherButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Weather</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="celsiusButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Celsius</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="fahrenheitButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Fahrenheit</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="kelvinButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Kelvin</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="humidText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pressureText">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>270</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="windSpeedText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="windDirText">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>320</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cityBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundPic">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>0</y>
      <width>711</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>seasons4 (1).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="sunriseText">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sourceText">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;All data sourced from &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;www.OpenWeatherMap.com&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sunsetText">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>380</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>backgroundPic</zorder>
   <zorder>tempText</zorder>
   <zorder>weatherButton</zorder>
   <zorder>celsiusButton</zorder>
   <zorder>fahrenheitButton</zorder>
   <zorder>kelvinButton</zorder>
   <zorder>humidText</zorder>
   <zorder>pressureText</zorder>
   <zorder>windSpeedText</zorder>
   <zorder>cityBox</zorder>
   <zorder>windDirText</zorder>
   <zorder>sunriseText</zorder>
   <zorder>sourceText</zorder>
   <zorder>sunsetText</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
